libraries/AP_GPS/GPS_Backend.h

```diff
@@ -53,8 +53,8 @@ class AP_GPS_Backend
     virtual void handle_msg(const mavlink_message_t *msg) { return ; }
     virtual void handle_gnss_msg(const AP_GPS::GPS_State &msg) { return ; }
 
-    // driver specific lag
-    virtual float get_lag(void) const { return 0.2f; }
+    // driver specific lag, returns true if the driver is confident in the provided lag
+    virtual bool get_lag(float &lag) const { lag = 0.2f; return true; }
```

Comment:
I happen to agree with you on the others (well and there is one I need to look at context wise to see who's right... I just had missed the review till today :)